docs/documentation.md

```diff
@@ -17,6 +17,111 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+# RabbitMQ Documentation
+
+Welcome to RabbitMQ documentation. Use the drop-down menu on the top right of any page to select the documentation version for the RabbitMQ release that you are using. We update each version of documentation with the latest minor and patch information for that release.
+
+For previous 3.12.x documentation, select the **3.12** option from the drop-down menu. 3.12.x documentation is hosted on our previous RabbitMQ site. This documentation is not versioned. 
+
+Use the navigation on the left to browse through documentation for your release of RabbitMQ.
+
+## Getting Started
+Choose which RabbitMQ version you want to use, review the [latest RabbitMQ releases](https://revamp-the-rabbitmq-home-pag.rabbitmq-website.pages.dev/docs/changelog) to help you decide. Next, go to the [Get Started pages](https://revamp-the-rabbitmq-home-pag.rabbitmq-website.pages.dev/tutorials) to try out the tutorials. These tutorials offer an introduction to messaging and it's key features as well as some common usage scenarios.
```

Comment:
We must use `./relative/paths/` here, we cannot be linking to `rabbitmq-website.pages.dev` as some of these will certainly be overlooked and deployed live. We've had it with website indexing, for example.

Reply:
Sure @michaelklishin will do, I am just using these links temporarily, thanks.